core/model/configuration.py

```diff
@@ -122,12 +123,32 @@ class ExternalIntegrationLink(Base, HasSessionCache):
     COLLECTION_MIRROR_SETTINGS = settings
 
 
+class ExternalIntegrationError(Base):
+    __tablename__ = "externalintegrationerrors"
+
+    id = Column(Integer, primary_key=True)
+    time = Column(DateTime, default=utc_now)
+    error = Column(Unicode)
+    external_integration_id = Column(
+        Integer,
+        ForeignKey(
+            "externalintegrations.id",
+            name="fk_error_externalintegrations_id",
+            ondelete="CASCADE",
+        ),
+    )
+
+
 class ExternalIntegration(Base):
 
     """An external integration contains configuration for connecting
     to a third-party API.
     """
 
+    class STATUS:
+        RED = 0
+        GREEN = 1
```

Comment:
Any reason not to just make this an enum?

Reply:
Laziness. This has been fixed. Not in me, in the code.

Reply:
I was reading through my GitHub notifications this morning while I was eating breakfast, and this comment gave me a good laugh 🤣